app/models/jobs_json_request_handler.rb

```diff
@@ -76,7 +76,8 @@ def convert_info(info_all, cluster)
         queue: j.queue_name,
         walltime_used: j.wallclock_time,
         username: j.job_owner,
-        extended_available: extended_available
+        extended_available: extended_available,
+        nodes: j.allocated_nodes.map{ |node| node.name }.reject(&:blank?)
```

Comment:
Need to also add `:allocated_nodes` to `JOB_ATTRS` so it is included in list of attributes requested for Info

Reply:
Add this now.